### PR TITLE
DP-98

### PR DESCRIPTION
**Pull request by drEfrati**
* **Main purpose**:
Fix the local date bug so now it should be in device's locale not whatever locale that is
 
* **Technical description**:
Use default local

* **Dilemmas you faced with?**

* **Tests**
only manually tests.

* **Documentation (Source/readme.md)**

### CODE REVIEW

File: kin-devplatform-core/src/main/java/kin/devplatform/core/util/DateUtil.java [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
 public class DateUtil {
 
-	private static DateFormat dateFormat = new SimpleDateFormat("dd.MM.yy", Locale.US);
 	private static DateFormat utcDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm", Locale.US);
+	private static DateFormat localFormat = DateFormat.getDateInstance(DateFormat.SHORT ,Locale.getDefault());
 	private static TimeZone utcTimeZone = TimeZone.getTimeZone("UTC");
 
 	public static String getDateFormatted(String dateStr) {
-		Date date = getDateFromUTCString(dateStr);
+;		Date date = getDateFromUTCString(dateStr);
```
Comment (yosriz) [inline kin-devplatform-core/src/main/java/kin/devplatform/core/util/DateUtil.java]: redundant `;` at line start

File: kin-devplatform-core/src/main/java/kin/devplatform/core/util/DateUtil.java [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
 		if (date != null) {
-			return dateFormat.format(date);
+			return localFormat.format(getDateFromUTCString(dateStr));
```
Comment (yosriz) [inline kin-devplatform-core/src/main/java/kin/devplatform/core/util/DateUtil.java]: probably using `getDateFromUTCString` here by mistake, should use date object

File: sample-app/build.gradle [hunk 3]
```diff
@@ -70,6 +70,11 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.6.2'
```
Comment (yosriz) [inline sample-app/build.gradle]: why LeakCanary is introduced on this PR?

**Comment by yosriz** [comment]
@drEfrati Please fill up PR template, and use "DP-98 - <bug title>" for the PR title and branch

File: sample-app/build.gradle (hunk 3 above)
Comment (drEfrati) [reply sample-app/build.gradle]: by mistake.
Should i remove it and add it it to a different pr?

Comment (yosriz) [reply sample-app/build.gradle]: yep

**Comment by drEfrati** [comment]
@yosriz 
Should i remove Leak Canary and add it to a different pr?

**Comment by drEfrati** [comment]
fix all comments

**Comment by drEfrati** [comment]
also removed leak canary.
You can the fixes

**Pull request closed without merging**